engines/engine.h

```diff
@@ -215,30 +291,37 @@ class Engine {
 	/** @name Overloadable methods
 	 *
 	 *  All Engine subclasses should consider overloading some or all of the following methods.
+	 *  @{
 	 */
-	//@{
 
+	/**
+	 * Initialize the OSystem for this engine.
```

Comment:
No.
It creates an instance of the Engine. The engine will be using the given OSystem instance.